lib/mappings/metadataMappings.ts

```diff
@@ -8,12 +49,18 @@ export const metadataMappings = {
     name: {
       type: 'keyword'
     },
+    objectValueList: {
```

Comment:
name suggestion: why not use plural instead of "List" ?
Like objectValues ? More concise

Reply:
I think it's important to know that meadata will be saved as a list.
If I write MetadataValues, the naïve representation is : 
{
  color : "red",
  size : 55
  ...
}
So my opinion is "list" in the name is an important information.